portal-api/src/generateApp.js

```diff
@@ -30,7 +30,7 @@ const generateApp = () => {
   app.use(createRateLimit());
   app.use(healthcheck);
   app.use(passport.initialize());
-  app.use(express.json());
+  app.use(express.json({ limit: '500kb' }));
```

Comment:
Sending the report data with the locations as structured JSON is quite big for large files. I spoke to Christian about this and he agreed for now we can up this limit to match the dtfs-central-api limit, but we might consider the priority or supporting even larger files and change how we send the data (perhaps zipping it) after discussing with product in a follow up PR

Reply:
Should this be in a constant?